SW4STM32/BellaLui/Application/HostBoard/Inc/misc/state_machine_helpers.h

```diff
@@ -0,0 +1,34 @@
+#ifndef MISC_STATE_MACHINE_HELPERS_H_
+#define MISC_STATE_MACHINE_HELPERS_H_
+
+#include <stdint.h>
+
+namespace state_machine_helpers {
+    // TODO: change to Enum
+    const uint8_t state_idle_false_positive = 11;
+    const uint8_t state_idle_liftoff_detected = 12;
+    const uint8_t state_idle_switch_to_liftoff_state = 13;
+
+    const uint8_t state_coast_rocket_is_ascending = 21;
+    const uint8_t state_coast_switch_to_primary_state = 22;
+
+    const uint8_t state_primary_altitude_above_secondary_altitude = 31;
+    const uint8_t state_primary_switch_to_secondary_state = 32;
+
+    const uint8_t state_secondary_altitude_difference_still_large = 41;
+    const uint8_t state_secondary_approaching_touchdown = 42;
+    const uint8_t state_secondary_switch_to_touchdown_state = 43;
+
+
+    uint8_t handleIdleState(const uint32_t currentTime, const uint32_t liftoff_time, const float acceleration_z);
+    bool handleLiftoffState(const uint32_t currentTime, const uint32_t previousTime);
+    uint8_t handleCoastState(const float max_altitude, const float baro_data_altitude, const float baro_data_base_altitude, const uint32_t apogee_counter);
```

Comment:
Not quite sure if the "base altitude" is correctly handled in the rest of the code... to be investigated whether it's needed and whether it can be used